### PR TITLE
Cookie updated by NetworkToCode Cookie Drift Manager Tool

### PR DESCRIPTION
**Pull request by snaselj**
Template:

```
{
    "template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
    "dir": "nautobot-app",
    "ref": "develop",
    "path": null
}
```

Cookie:

```
{
    "remote": "https://github.com/nautobot/nautobot-plugin-floor-plan.git",
    "path": "/opt/ntc/drift-manager/outputs/nautobot-plugin-floor-plan",
    "repository_path": "/opt/ntc/drift-manager/outputs/nautobot-plugin-floor-plan",
    "dir": "",
    "branch_prefix": "drift-manager",
    "context": {
        "codeowner_github_usernames": "\\",
        "full_name": "Network to Code, LLC",
        "email": "info@networktocode.com",
        "github_org": "nautobot",
        "plugin_name": "nautobot_floor_plan",
        "verbose_name": "Nautobot Floor Plan",
        "plugin_slug": "nautobot-floor-plan",
        "project_slug": "nautobot-plugin-floor-plan",
        "repo_url": "https://github.com/nautobot/nautobot-plugin-floor-plan",
        "base_url": "floor-plan",
        "min_nautobot_version": "2.0.0",
        "max_nautobot_version": "2.9999",
        "camel_name": "FloorPlan",
        "project_short_description": "Nautobot Floor Plan",
        "model_class_name": "FloorPlan",
        "open_source_license": "Apache-2.0",
        "docs_base_url": "https://docs.nautobot.com",
        "docs_app_url": "https://docs.nautobot.com/projects/floor-plan/en/latest",
        "_template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
        "_output_dir": "/opt/ntc/drift-manager/outputs",
        "_repo_dir": "/opt/ntc/drift-manager/outputs/.cookiecutters/cookiecutter-nautobot-app/nautobot-app",
        "_checkout": "develop"
    },
    "base_branch": "develop",
    "remote_name": "origin",
    "pull_request_strategy": "PullRequestStrategy.CREATE",
    "post_actions": [
        "PostAction.BLACK"
    ],
    "baked_commit_ref": "",
    "draft": true
}
```

CLI Arguments:

```
{
    "cookie_dir": "",
    "input": false,
    "json_filename": "setup-cookie-floor-plan.json",
    "output_dir": "./outputs",
    "push": true,
    "template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
    "template_dir": "nautobot-app",
    "template_ref": "develop",
    "pull_request": null,
    "post_action": [
        "black"
    ],
    "disable_post_actions": false,
    "draft": true
}
```

### CODE REVIEW

**Comment by snaselj** [comment]
This PR is ready for review now.

cc.: @whitej6

**Review by whitej6 — APPROVED** [review]

File: development/nautobot_config.py [hunk 1]
```diff
@@ -43,20 +59,28 @@
...
+        "OPTIONS": {
+            "CLIENT_CLASS": "django_redis.client.DefaultClient",
+        },
+    }
+}
 
-# Django Debug Toolbar
-DEBUG_TOOLBAR_CONFIG = {"SHOW_TOOLBAR_CALLBACK": lambda _request: DEBUG and not TESTING}
+# Redis Cacheops
+CACHEOPS_REDIS = parse_redis_connection(redis_database=1)
```
Comment (glennmatthews) [inline development/nautobot_config.py:78]: Shouldn't be needed in a Nautobot-2.0-required version of the plugin.

Comment (whitej6) [reply development/nautobot_config.py:78]: Good catch, we'll fix in the cookie and will get picked up in the next round be drift manager.

Comment (snaselj) [reply development/nautobot_config.py]: Removed

**Pull request merged**